File: auto-docs/executables/python/axes_reversed.py
# Learn about API authentication here: plot.ly/python/getting-started
# Find your api_key here: plot.ly/settings/api

import plotly.plotly as py
from plotly.graph_objs import *

py.sign_in('TestBot', 'r1neazxo9w')
data = Data([
    Scatter(
        x=[1, 2],
        y=[1, 2]
    )
])
layout = Layout(
    xaxis=XAxis(
        autorange='reversed'
    )
)
fig = Figure(data=data, layout=layout)
plot_url = py.plot(fig, filename='axes-reversed', auto_open=False)

File: auto-docs/executables/node/axes_reversed.js
var plotly = require('plotly')('TestBot', 'r1neazxo9w')

var data = [
  {
    x: [1, 2], 
    y: [1, 2], 
    type: "scatter"
  }
];
var layout = {xaxis: {autorange: "reversed"}};

var graph_options = {filename: "axes-reversed", fileopt: "overwrite", layout: layout, auto_open: "false"}
plotly.plot(data, graph_options, function (err, msg) {
    console.log(msg);
});